scripts/performance-tests-kevm.sh

```diff
@@ -54,18 +54,45 @@ fi
 
 git submodule update --init --recursive --depth 1 kevm-pyk/src/kevm_pyk/kproj/plugin
 
+BUG_REPORT=''
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --bug-report)
+      mkdir -p $SCRIPT_DIR/bug-reports/kevm-$KEVM_VERSION-$FEATURE_BRANCH_NAME
+      BUG_REPORT="--bug-report --bug-report-dir $SCRIPT_DIR/bug-reports/kevm-$KEVM_VERSION-$FEATURE_BRANCH_NAME"
+      shift # past argument
+      ;;
+    -*|--*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      shift # past argument
+      ;;
+  esac
+done
```

Comment:
:+1: good idea.
but let's not make it swallow unknown arguments (please remove the `*)` case).
We could move this to the start and make it read `KEVM_VERSION` and `PYTEST_PARALLEL` (which we should default to 3 instead of 3 while at it.